core/silentlogger/module.go

```diff
@@ -0,0 +1,71 @@
+package silentlogger
+
+import (
+	"context"
+
+	"flamingo.me/dingo"
+
+	"flamingo.me/flamingo/v3/framework/flamingo"
+)
+
+type (
+	// Module for Silent ZAP logging
+	Module struct{}
+
+	shutdownEventSubscriber struct {
+		logger flamingo.Logger
+	}
+)
+
+// Configure the ZAP logger as flamingo.Logger (in JSON mode kibana compatible)
+func (m *Module) Configure(injector *dingo.Injector) {
+	registry := new(LoggingContextRegistry)
```

Comment:
This is not necessary, rather reference them in the bindings `Bind(new(LoggingContextRegistry))`...

Reply:
in line 27 I used this reference in ```flamingo.BindEventSubscriber(injector).To(registry)```, will EagerSingleton still be referenced if I create two instances?